tools/ExtensionLibrary/DevHome.ExtensionLibrary/Views/ExtensionLibraryView.xaml

```diff
@@ -74,7 +74,7 @@
                                     <!-- This StackPanel is a workaround for the bug https://github.com/CommunityToolkit/Windows/issues/396 -->
                                     <Button x:Uid="MoreOptionsButton"
                                             Content="&#xe712;"
-                                            Height="36" Width="36"
+                                            Height="36" MinWidth="36"

```

Comment:
is Setting minWidth the correct solution?  Would getting rid of Width be better?  Allow the content to expand and shrink dynamically?  With MinWIdth, the content won't shrink past a certain point.